BeatTogether.DedicatedServer.Kernel/Managers/LobbyManager.cs

```diff
@@ -318,27 +265,90 @@ private void UpdateBeatmap(BeatmapIdentifier? beatmap, GameplayModifiers modifie
             }
         }
 
+        // Sets countdown and beatmap how the client would expect it to
+        // If you want to cancel the countdown use CancelCountdown(), Not SetCountdown as CancelCountdown() ALSO informs the clients it has been canceled
+        public void SetCountdown(CountdownState countdownState, float countdown = 0)
+        {
```

Comment:
New function that handles setting,resetting,sending beatmap and countdown using new enum.